lgn-worker/src/one-shot.rs

```diff
@@ -73,10 +73,7 @@ async fn main() -> Result<()> {
     config.validate();
     let checksums = fetch_checksums(config.public_params.checksum_file_url()).await?;
 
-    let provers_manager = tokio::task::block_in_place(move || -> Result<ProversManager> {
-        ProversManager::new(&config, &checksums).context("while registering provers")
-    })
-    .context("creating prover managers")?;
```

Comment:
This was needed because of the `reqwest::blocking::Client`. After this PR it is no longer used.